### DISCUSSION TITLE
Send and receive SMS on MAUI

### DISCUSSION THREAD

**Discussion by behroozbc** [dotnet/maui#3606] [Q&A] [answered]
hi 
I am developing MAUI project
My app need Send SMS to phone number and get received message form same phone number

Is possible to do this in maui? And how can implement ?

**Comment by nogginbox** [comment, marked as answer]
There is an Sms class inside Microsoft.Maui.Essentials and it seems to work the same way it did in Xamarin.Essesentials.

There is an [example of the Xamarin essentials version on C# Corner](https://www.c-sharpcorner.com/article/xamarin-forms-send-sms-using-xamarin-essentials/).

This is for sending text messages. I'm not sure you can receive on Android and IoS. You definitely can't on IoS due to restrictions on the device.
